samples/pom.xml

```diff
@@ -33,7 +33,18 @@
   <packaging>jar</packaging>
 
   <name>Apache Apex Malhar (incubating) Samples</name>
-
+  <build>
```

Comment:
Adding "checkstyle.skip" under properties does not work?

https://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html